test/plugin.spec.js

```diff
@@ -93,7 +93,7 @@ describe('Babel Root Import - Plugin', () => {
   });
 
   it('supports re-export syntax (export * from ... or export { named } from ...)', () => {
-    const targetRequire = slash(`${process.cwd()}/some/example.js`);
+    const targetRequire = slash('/some/example.js');
     const transformedExportNamed = babel.transform("export * from '@/some/example.js';", {
```

Comment:
I think you named the functions in the wrong order:
`transformedExportNamed` makes the `"export * from '@/some/example.js';`
and the
`transformedExportAll` makes `export { named } from ...`